lib/install/elm.rb

```diff
@@ -0,0 +1,22 @@
+require "webpacker/configuration"
+
+puts "Copying elm loader to #{Webpacker::Configuration.config_path}/loaders"
+copy_file "#{__dir__}/config/loaders/installers/elm.js",
+          "#{Webpacker::Configuration.config_path}/loaders/elm.js"
+
+puts "Copying elm example entry file to #{Webpacker::Configuration.entry_path}"
+copy_file "#{__dir__}/examples/elm/Main.elm", "#{Webpacker::Configuration.entry_path}/Main.elm"
+
+puts "Copying elm app file to #{Webpacker::Configuration.entry_path}"
+copy_file "#{__dir__}/examples/elm/hello_elm.js",
+          "#{Webpacker::Configuration.entry_path}/hello_elm.js"
+
+puts "Updating .gitignore to include elm-stuff folder"
+insert_into_file ".gitignore", "/elm-stuff\n", before: "/node_modules\n"
+
+puts "Installing all elm dependencies"
+run "./bin/yarn add elm"
+run "./bin/yarn add --dev elm-hot-loader elm-webpack-loader"
+run "elm package install --yes"
```

Comment:
Good catch, thanks. Too used to working with my current configuration and wasn't thinking about those that might not have installed Elm via `brew install elm`.

[Fixed](https://github.com/bkuhlmann/webpacker/blob/elm/lib/install/elm.rb#L20) (via rebase). Notice that I've prefixed the command with `yarn run` since the Elm installer has Yarn install Elm prior to this command as a package dependency. The use of `--` before `--yes` is to ensure that option gets passed to Elm and not Yarn so that Elm won't prompt anyone to confirm the Elm packages being installed (in order to keep this process non-interactive).